barretenberg/cpp/src/barretenberg/execution_trace/execution_trace.hpp

```diff
@@ -41,8 +41,9 @@ template <class Flavor> class ExecutionTrace_ {
      * @brief Given a circuit, populate a proving key with wire polys, selector polys, and sigma/id polys
      *
      * @param builder
+     * @param is_structured whether or not the trace is to be structured with a fixed block size
```

Comment:
I think this is a good place to explain why the trace would need to be structured, just a concise sentence should be fine

Reply:
good idea, done